pom.client.xml

```diff
@@ -419,6 +419,31 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>net.jonathangiles.tools</groupId>
+        <artifactId>whitelistgenerator-maven-plugin</artifactId>
+        <version>1.0.1</version> <!-- {x-version-update;net.jonathangiles.tools:whitelistgenerator-maven-plugin;external_dependency} -->
+        <executions>
+          <execution>
+            <id>generateWhitelistReport</id>
+            <phase>package</phase>
+            <goals>
+              <goal>report</goal>
+            </goals>
+            <configuration>
+              <baseDir>.</baseDir>
+              <reportFile>dependency-whitelist.json</reportFile>
+              <ignoredGA>
+                <param>com.azure:azure-cosmos-examples</param>
+                <param>com.azure:azure-cosmos-benchmark</param>
+                <param>com.azure:azure-core-test</param>
+                <param>com.azure:azure-e2e</param>
```

Comment:
You're going to want to add com.azure:azure-storage-perf and com.azure:perf-test-core to this list. Right now, they're not being built as part of any pipeline but when I do the versioning work for the whitelist includes they will be.

Reply:
Added both these modules to ignored list.